<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vaccineRecordDialog</class>
 <widget class="QDialog" name="vaccineRecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vaccine record</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Vaccine record update page</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="inputDate">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>User ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="userIdLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Vaccineted date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="vaccineDateEdit">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Vaccine record</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QRadioButton" name="firstRadioButton">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>1st Vaccine</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="secondRadioButton">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>2nd Vaccine</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
